projects/components/src/stepper/tab-controls/stepper-tab-controls.component.ts

```diff
@@ -0,0 +1,23 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../../content/content-holder';
+import { StepperComponent } from '../stepper.component';
+import { StepperTabComponent } from '../tab/stepper-tab.component';
+
+@Component({
+  selector: 'ht-stepper-tab-controls',
+  template: CONTENT_HOLDER_TEMPLATE,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StepperTabControlsComponent extends ContentHolder {
+  public constructor(public stepper: StepperComponent, public step: StepperTabComponent) {
+    super();
+  }
+
+  public next(): void {
+    this.stepper?.next();
+  }
+
+  public previous(): void {
+    this.stepper?.previous();
+  }
```

Comment:
I believe provided templates should call these methods. Need to find a way to mandate that for a user?

Reply:
These are some basic methods that the custom control would want to call, so we expose them. The consumer could do this:
```html
<ht-stepper-tab-controls #controls> <!-- Get access to component instance -->
      <div>
        <div class="back-next">
          <ht-button (click)="controls.stepper.previous()"></ht-button> <!-- This here on click -->
        </div>
      </div>
    </ht-stepper-tab-controls>
```

Reply:
What happens if we have two tabs defined `tab1`, `tab2` and two set of controls `#controls1`, `#controls2`. 
Now, active tab is `tab1` but user calls `#controls2`?

Reply:
I believe we should expose `next/back` controls from a single place i.e. from `StepperComponent` instead of tab level i.e. `StepperTabComponent`.

Reply:
For any handling on the parent we can emit events

Reply:
It is all referencing the same `stepper` instance. So nothing could go wrong here, it just for convenience.

Reply:
> What happens if we have two tabs defined `tab1`, `tab2` and two set of controls `#controls1`, `#controls2`. Now, active tab is `tab1` but user calls `#controls2`?

Accessing same instance from different places can get cumbersome? Component should expose only one API for a single type of action.

Reply:
Stepper instance inside controls is made private now.